src/api.js

```diff
@@ -10,3 +10,9 @@ export const getAllArticles = (slug) => {
 export const getTopics = () => {
   return axios.get("https://leonie-phelps-nc-news-be.herokuapp.com/api/topics");
 };
+
+export const getArticleById = (article_id) => {
+  return axios.get(
+    `https://leonie-phelps-nc-news-be.herokuapp.com/api/articles/${article_id}`
+  );
+};
```

Comment:
`axios.create` might help make your life easier in terms of writing out the full URL in each of these requests 🙂